UndertaleModLib/AumiIPC.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UndertaleModLib
+{
+    public struct IpcMessage_t
+    {
+        public short FuncID;
+        [MarshalAs(UnmanagedType.ByValArray, SizeConst = 510)]
+        public byte[] Buffer;
+
+        public byte[] RawBytes()
+        {
+            var stream = new MemoryStream();
+            var writer = new BinaryWriter(stream);
```

Comment:
I think you might want `using` here, otherwise these might not get cleaned up properly.

Reply:
Pushed a fix.